File: java-bench/src/main/java/bench/LoopBench.java
package bench;

public class LoopBench {

    private static final int N = 100_000_000;

    public static void main(String[] args) {
        long start = System.currentTimeMillis();
        long sum = 0;
        for (int i = 0; i < N; i++) {
            sum += i;
        }
        long end = System.currentTimeMillis();
        System.out.println("result: " + sum);
        System.out.println("elapsed time: " + (end - start) + " ms");
    }
}

File: csharp-bench/LoopBench.cs
using System;
using System.Diagnostics;

class LoopBench
{
    const long N = 1_000_000_000;

    static void Main()
    {
        var sw = Stopwatch.StartNew();
        long sum = 0;
        for (long i = 0; i < N; i++)
        {
            sum += i;
        }
        sw.Stop();
        Console.WriteLine($"result: {sum}");
        Console.WriteLine($"elapsed time: {sw.ElapsedMilliseconds} ms");
    }
}
